app/controllers/admin/settings/billing.js

```diff
@@ -0,0 +1,21 @@
+import Controller from '@ember/controller';
+import { timezones } from 'open-event-frontend/utils/dictionary/date-time';
+import { computed } from '@ember/object';
+
+export default Controller.extend({
```

Comment:
Lets start moving to ES6 style classes for all new resources. 
https://github.com/fossasia/open-event-frontend/pull/3169/files